mmv1/products/vertexai/FeatureOnlineStoreFeatureview.yaml

```
@@ -118,3 +124,62 @@ properties:
         description: |
           Columns to construct entityId / row keys. Start by supporting 1 only.
         item_type: Api::Type::String
+  - !ruby/object:Api::Type::NestedObject
+    name: 'vectorSearchConfig'
+    description: |
+      Configuration for vector search. It contains the required configurations to create an index from source data, so that approximate nearest neighbor (a.k.a ANN) algorithms search can be performed during online serving.
+    immutable: true
+    min_version: beta
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'embeddingColumn'
+        required: true
+        description: |
+          Column of embedding. This column contains the source data to create index for vector search. embeddingColumn must be set when using vector search.
+      - !ruby/object:Api::Type::Array
+        name: 'filterColumns'
+        description: |
+          Columns of features that're used to filter vector search results.
```

nit: not a blocker but "that're" is a bit clunky and could make sense to expand (though this matches the API docs). https://developers.google.com/style/contractions

```suggestion
          Columns of features that are used to filter vector search results.
```